### PR TITLE
#1426 Import sequence format for RNA, DNA and PEPTIDES

### PR DESCRIPTION
**Pull request by even1024**

### CODE REVIEW

File: api/c/indigo/src/indigo_layout.cpp [hunk 1]
```diff
@@ -48,48 +49,56 @@ CEXPORT int indigoLayout(int object)
...
             {
-                ml.filter = &f;
+                SequenceLayout sl(*mol);
+                sl.make();
             }
-
-            ml.max_iterations = self.layout_max_iterations;
-            ml.bond_length = MoleculeLayout::DEFAULT_BOND_LENGTH;
-            ml.layout_orientation = (layout_orientation_value)self.layout_orientation;
-            if (self.layout_preserve_existing || mol->hasAtropoStereoBonds())
```
Comment (AliaksandrDziarkach) [inline api/c/indigo/src/indigo_layout.cpp:61]: Looks like you accidentally delete option "layout_preserve_existing"

Comment (even1024) [reply api/c/indigo/src/indigo_layout.cpp:61]: fixed

File: core/indigo-core/molecule/src/monomer_commons.cpp [hunk 2]
```diff
@@ -44,18 +30,54 @@ namespace indigo
...
-            auto it = kAliasToAminoAcid.find(alias);
-            return it != kAliasToAminoAcid.end();
-        }
-        return false;
+        return isAminoAcidClass(monomer_class) && monomerNameByAlias(monomer_class, alias).size() != alias.size();
     }
 
     std::string monomerNameByAlias(const std::string& monomer_class, const std::string& alias)
     {
+        static const std::unordered_map<std::string, std::string> kAliasToNucleic = {{"A", "Ade"}, {"C", "Cyt"},  {"G", "Gua"},  {"U", "Ura"}, {"T", "Thy"},
```
Comment (AliaksandrDziarkach) [inline core/indigo-core/molecule/src/monomer_commons.cpp:74]: Looks like some alias missed - kNucleicToAlias contains more elements.

Comment (even1024) [reply core/indigo-core/molecule/src/monomer_commons.cpp:74]: fixed

**Review by AliaksandrDziarkach — APPROVED** [review]
LGTM

**Pull request merged**